lib/src/components/NxTableHead/NxTableHead.tsx

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import {addPropsToChildren} from '../../util/childUtil';
+
+import { Props } from './types';
+export {Props} from './types';
+
+const NxTableHead = function NxTableHead(props: Props) {
+  const {className, children, ...attrs} = props;
+
+  return (
+    <thead className={className} {...attrs}>
+      {addPropsToChildren(children, {isHeader: true})}
```

Comment:
What's the use-case for this `addPropsToChildren` fn?

Reply:
Both `NxTableHead` and `NxTableRow` use it to do automatic prop-drilling for the `isHeader` prop. This means you just use `NxTableCell` without having to pass the isHeader prop to it to get the correct css classes applied. I added that function to reduce the duplicated code and it felt potentially useful in other situations where we want a component to apply arbitrary props to children.